data/mods/Magiclysm/monsters/monsters.json

```diff
@@ -328,7 +328,101 @@
     "harvest": "lemure",
     "anger_triggers": [ "PLAYER_CLOSE" ],
     "death_function": [ "NORMAL" ],
-    "regenerates": 1,
-    "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE", "REVIVES" ]
+    "flags": [ "SEES", "SMELLS", "HEARS", "STUMBLES", "PATH_AVOID_FIRE", "REGENERATES_1", "REVIVES" ]
```

Comment:
this looks like a rebase error

Reply:
It happens.

Reply:
![image](https://user-images.githubusercontent.com/30208401/70840370-c3510200-1ddf-11ea-8603-8e3756908be0.png)

Reply:
I finally noticed that "regenerates" flag doesn't exist anymore.